src/libs/tmdk/src/commands/deploy.ts

```diff
@@ -112,12 +112,29 @@ export const handler = async (argv: Arguments<Options>) => {
         ).reduce((acc, [key, value]) => {
           if (!value["isInherited"]) {
             acc[key] = value;
+          } else if (value["defaultValue"] != undefined) {
```

Comment:
Nit: As with the previous pr, we should prefer the more idiomatic style here:
https://stackoverflow.com/questions/20804163/check-if-a-key-exists-inside-a-json-object
i.e. `!("defaultValue" in value)`